src/aspnetcore/eng/DotNetBuild.props

```diff
@@ -1,78 +0,0 @@
-<!-- When altering this file, please include @dotnet/product-construction as a reviewer. -->
-
-<Project>
-
-  <PropertyGroup>
-    <GitHubRepositoryName>aspnetcore</GitHubRepositoryName>
-    <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
-  </PropertyGroup>
-
-  <!--
-    Remove inner source .globalconfig file as both the inner and outer config files get loaded and cause a conflict.
-    Leaving the inner will cause all conflicting settings to be ignored.
-    https://learn.microsoft.com/dotnet/fundamentals/code-analysis/configuration-files#general-options.
-    This only needs to be done if there is an inner clone.
-  -->
-  <Target Name="RemoveInnerGlobalConfig"
-          DependsOnTargets="PrepareInnerSourceBuildRepoRoot"
-          BeforeTargets="RunInnerSourceBuildCommand"
-          Condition="'$(DotNetBuildOrchestrator)' != 'true'">
-
-    <Delete Files="$(InnerSourceBuildRepoRoot).globalconfig" />
-  </Target>
-
-  <!-- Build RepoTasks - this is normally triggered via the build script but the inner ArPow source-build is run via msbuild.
-       https://github.com/dotnet/source-build/issues/3807 -->
-  <Target Name="BuildRepoTasks"
-          DependsOnTargets="GetSourceBuildCommandConfiguration"
-          Condition="'$(DotNetBuildInnerRepo)' == 'true' and '$(DotNetBuildSourceOnly)' == 'true'"
-          BeforeTargets="Execute">
-    <!-- If the alternative runtime location and key are present, pass those through -->
-    <PropertyGroup>
-      <_AdditionalRepoTaskBuildArgs />
-      <_AdditionalRepoTaskBuildArgs Condition="'$(DotNetRuntimeSourceFeed)' != ''" >$(_AdditionalRepoTaskBuildArgs) --runtimesourcefeed $(DotNetRuntimeSourceFeed)</_AdditionalRepoTaskBuildArgs>
-      <_AdditionalRepoTaskBuildArgs Condition="'$(DotNetRuntimeSourceFeedKey)' != ''" >$(_AdditionalRepoTaskBuildArgs) --runtimesourcefeedkey $(DotNetRuntimeSourceFeedKey)</_AdditionalRepoTaskBuildArgs>
-      <_AdditionalRepoTaskBuildArgs Condition="'$(RestoreConfigFile)' != ''" >$(_AdditionalRepoTaskBuildArgs) /p:RestoreConfigFile=$(RestoreConfigFile)</_AdditionalRepoTaskBuildArgs>
-    </PropertyGroup>
-
-    <ItemGroup>
-      <!-- We need to flow FullAssemblySigningSupported even when building repo tasks because they use full signing -->
-      <InnerBuildEnv Condition="'$(FullAssemblySigningSupported)' != ''" Include="FullAssemblySigningSupported=$(FullAssemblySigningSupported)" />
-    </ItemGroup>
-
-    <!-- Call the build.sh script to build the repo tasks. Set IgnoreStandardErrorWarningFormat
-         to true. This avoids fatal errors, because in internal builds there are usually a few failed installation
-         attempts as the install script walks through potential locations for a runtime.
-         The overall build script will return a proper exit code, but we don't want to pick up the printed error messages. -->
-    <Exec
-      Command="./eng/build.sh --only-build-repo-tasks -bl $(_AdditionalRepoTaskBuildArgs) $(InnerBuildArgs)"
```

Comment:
@wtgodbe - IIRC this option was added to support this scenario.  If that is the case perhaps this option should be cleaned up to help simplify the script.  I'll let you follow-up as necessary.